agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -17,30 +17,75 @@
 import com.google.gson.Gson;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Arrays;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /**
- * Servlet that returns some example content.
- * TODO: modify this file to handle comments data
+ * Servlet that accepts data from the comments form &
+ * posts that data to the comments.html page
  */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private class Comment {
```

Comment:
I like how you encapsulated this data within an object!

Reply:
Thank you!